### ISSUE TITLE
[epic][docs] accessibility tab

### ISSUE THREAD

**Issue by nikkimk** [RedHat-UX/red-hat-design-system#1296]
Verify accessibility tab of each component has identified correct WCAG Guidelines, and that guidelines section has proper spacing. For example, tag guidelines don't include use of color but should, and table has extra spaces between the guideline bullets.

List of RH elements and missing WCAG success criteria:

- [ ] **Accordion**
  - [ ] 1.3.2 Meaningful Sequence — Level A
  - [ ] 1.4.1 Use of Color — Level A
  - [ ] 1.4.3 Contrast (Minimum) — Level AA
  - [ ] 2.4.6 Headings and Labels — Level AA
  - [ ] 2.4.7 Focus Visible — Level AA
  - [ ] 2.4.10 Section Headings — Level AAA
  - [ ] 3.2.4 Consistent Identification — Level AA
- [ ] **Alert**
  - [ ] 1.3.1 Info and Relationships — Level A
  - [ ] 1.4.1 Use of Color — Level A
  - [ ] 1.4.3 Contrast (Minimum) — Level AA
  - [ ] 2.4.6 Headings and Labels — Level AA
  - [ ] 3.3.1 Error Identification — Level A
  - [ ] 4.1.3 Status Messages — Level AA
- [ ] **Audio player**
  - [ ] 1.4.2 Audio Control — Level A
  - [ ] 3.2.4 Consistent Identification — Level AA
- [ ] Avatar
- [ ] Badge
- [ ] Blockquote
- [ ] Call to action
- [ ] Card
- [ ] Code block
- [ ] Dialog
- [ ] Footer
- [ ] Jump links
- [ ] Navigation, primary
- [ ] Navigation, secondary
- [ ] Pagination
- [ ] Popover
- [ ] Progress steps
- [ ] Spinner
- [ ] Statistic
- [ ] Subnavigation
- [ ] Table
- [ ] Tabs
- [ ] Tag
- [ ] Tile
- [ ] Timestamp
- [ ] Tooltip

## Steps

1. Go to each element's accessibility tab
2. Check the WCAG Guidelines bulleted section
3. Guidelines should be applicable to specific component and are spaced properly
